app/src/features/projects/view/components/ProjectObjectives.tsx

```diff
@@ -99,6 +100,58 @@ const ProjectObjectives: React.FC<IProjectObjectivesProps> = (props: any) => {
               </>
             )}
           </Grid>
+
+          <Grid container item xs={12} spacing={3} justify="space-between" alignItems="center">
+            <Grid item>
+              <Typography variant="h3">Project Caveats</Typography>
+            </Grid>
+          </Grid>
+          <Grid container item spacing={2} xs={12}>
+            {isTruncatedCaveats && (
+              <>
+                <Grid item xs={12}>
+                  {objectives.caveats
+                    .slice(0, determineTruncatingLength())
+                    .split('\n')
+                    .map((paragraph: string) => {
+                      if (paragraph) {
+                        return (
+                          <Typography style={{ wordBreak: 'break-all' }} key={uuidv4()}>
+                            {paragraph}
+                          </Typography>
+                        );
+                      }
+                      return <p key={uuidv4()}></p>;
```

Comment:
we need a test case where we are rendering a long caveat and a long objective string with multilines - so that this line and the similar line in the objectives section are covered in testing

also i think this entire section of similar code should be turned into a component where we just pass in whether it is dealing with caveats or objectives and it renders the desired JSX for us - rather than writing it twice

Reply:
done